awx/ui/client/features/jobs/routes/jobs.route.js

```diff
@@ -15,6 +15,7 @@ export default {
         job_search: {
             value: {
                 not__launch_type: 'sync',
+                not__type: 'workflow_approval',
```

Comment:
@mabashian do we still need this?  I _thought_ we updated the API to exclude this.

Reply:
We sure don't - I'll remove